slides-hpccarpentry.md

```diff
@@ -112,28 +121,25 @@ from the 2020 [carpentries long-term assessment report](https://carpentries.gith
 
 ---
 
+# What HPCCarpentry is not?
```

Comment:
I hesitated to include this when I created my draft agenda. What I was thinking here was that people would approach HPC Carpentry expecting it to be the source of training content for all things HPC related.

I was thinking of including an image from https://app.diagrams.net/?client=1&lightbox=1&edit=_blank to get this across. That this is more about scalable training to make HPC accessible to the wider scientific community.

Reply:
sounds good! Go for it.